packages/frontend/src/components/Task/style.ts

```diff
@@ -3,7 +3,7 @@ import { createStyles } from '@mantine/core'
 const useStyles = createStyles((theme) => ({
   taskMain: {
     borderRadius: theme.radius.md,
-    boxShadow: '0px 4px 12px 0px #0000000D',
```

Comment:
From what i remember it's copy paste from Figma, what's the reason for changing the value?

Reply:
I think Figma isn't showing the real values, because in reality it's practically invisible.

Reply:
As far as I know figma presents designs one to one as it is web based. After taking a look in my opinion it looks like it because we have pure white background, not the same as on te design itself